### DISCUSSION TITLE
About DuckDB database show BLOB type error

### DISCUSSION THREAD

**Discussion by EastLord** [dbeaver/dbeaver#35346] [DBeaver Development]
https://github.com/dbeaver/dbeaver/issues/35328
I want to fix this bug where DBeaver fails to show BLOB data in a table from a DuckDB database, but I encountered a bit of an issue.
Here is the code I modified.
<img width="1381" alt="image" src="https://github.com/user-attachments/assets/eceb0724-b772-4ee2-9f60-b91b45524010">
<img width="1381" alt="image" src="https://github.com/user-attachments/assets/6e6e3053-99e5-4e66-a22c-15db421193e8">
I noticed that the value is incorrect.
<img width="805" alt="image" src="https://github.com/user-attachments/assets/17e8cae6-b078-4813-8bbb-a09bf3e1c5fe">

**Comment by cloudcell** [comment]
I can confirm that previously I experienced a similar behavior from this plugin. When I refreshed the view, the first record would give different values. Perhaps the issue is deeper.

**Reply by cloudcell** [reply]
UPDATE: perhaps this thread might be also helpful: https://github.com/duckdb/duckdb-java/issues/70
